deepchem/feat/molecule_featurizers/roberta_tokenizer.py

```diff
@@ -0,0 +1,39 @@
+from deepchem.feat import MolecularFeaturizer
+from transformers import RobertaTokenizerFast
+from deepchem.utils.typing import RDKitMol
+
+from typing import Dict, List
+
+
+class RobertaFeaturizer(RobertaTokenizerFast, MolecularFeaturizer):
```

Comment:
Can you add a docstring here explaining what the class does along with a usage example?